third_party/gpus/cuda/platform.bzl

```diff
@@ -0,0 +1,39 @@
+CUDA_VERSION = ""
+CUDNN_VERSION = ""
+PLATFORM = ""
+
+def cuda_sdk_version():
+  return CUDA_VERSION
+
+def cudnn_sdk_version():
+  return CUDNN_VERSION
+
+def cuda_library_path(name, version = cuda_sdk_version()):
+  if PLATFORM == "Darwin":
+    return "lib/lib{}.{}.dylib".format(name, version)
+  else:
+    return "lib64/lib{}.so.{}".format(name, version)
```

Comment:
I think the problem is that you are assuming the version is always set -- but we allow people to use the unversioned library (libfoo.so, not libfoo.so.version).

I'll kick off a test just to make sure, but I still think this may not work.